themes/sdss_subtheme/src/scss/components/lockup/_lockup.scss

```
@@ -1,5 +1,25 @@
 @charset "UTF-8";
 
+// Logo
+.su-masthead {
+
+  .su-lockup {
+    @include grid-media-min('lg') {
+      margin-bottom: 0;
+    }
+
+    .su-lockup__custom-logo {
+      @include grid-media-min('lg') {
+        max-height: 5.5rem;
+      }
+
+      @include grid-media-max('md') {
+        max-height: 5rem;
+      }
```

```suggestion
    .su-lockup__custom-logo {
      max-height: 5rem;
      
      @include grid-media-min('lg') {
        max-height: 5.5rem;
      }
```

Why not something like this?

---

Yes I like that idea better @joegl